ultra/tests/test_ultra_package.py

```diff
@@ -0,0 +1,88 @@
+# MIT License
+#
+# Copyright (C) 2021. Huawei Technologies Co., Ltd. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+import os, shutil, ray
+import unittest
+from ultra.train import train
+from ultra.scenarios.generate_scenarios import build_scenarios
+
+
+class UltraPackageTest(unittest.TestCase):
+    def test_simple_train_run(self):
+        save_dir = "tests/task/eval_test/"
+        log_dir = "tests/logs"
+
+        if os.path.exists(save_dir):
+            shutil.rmtree(save_dir)
+
+        save_dir = "tests/task/eval_test/eval"
+
+        try:
+            build_scenarios(
+                task="task00",
+                level_name="eval_test",
+                stopwatcher_behavior=None,
+                stopwatcher_route=None,
+                root_path="tests/scenarios",
+                save_dir=save_dir,
+            )
+        except:
+            assert False
```

Comment:
Not sure how necessary this is, but should we use PyTest's assertions for the assertions in this test? For example:
```suggestion
            self.assertTrue(False)
```
The same can be done with the other assertions in this test. I just noticed that the PyTest assertions were used in all other ULTRA tests. Let me know your thoughts.